src/QueryType/Select/Query/FilterQuery.php

```diff
@@ -136,6 +136,58 @@ public function setTags(array $tags): self
         return $this;
     }
 
+    /**
+     * Cache the filter query or not.
+     *
+     * @param bool $cache
+     *
+     * @return self Provides fluent interface
+     */
+    public function setCache(bool $cache): self
+    {
+        $this->getLocalParameters()->setCache($cache);
+
+        return $this;
+    }
+
+    /**
+     * Get the information if the filter query should be cached or not.
+     *
+     * @return bool
+     */
+    public function getCache(): bool
+    {
+        $cache = $this->getLocalParameters()->getCache();
+        // The default is to cache the filter Query.
+        return [] === $cache | 'true' === $cache[0];
```

Comment:
needs to use `||` or it will fail for empty arrays

Reply:
I had to push without local testing because I'm currently upgrading PHP.

Reply:
I implemented it differently now.